common/server/statshouse/add_metrics_batch.cpp

```diff
@@ -0,0 +1,61 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include "add_metrics_batch.h"
+#include <common/tl/methods/string.h>
+
+#include <utility>
+
+#include "common/tl/constants/statshouse.h"
+#include "common/tl/parse.h"
+#include "common/tl/store.h"
+
+namespace statshouse {
+
+constexpr int BUFFER_LEN = 1000;
+
+static int buffer[BUFFER_LEN];
```

Comment:
Also I think that `buffer` should be managed by `StatshouseClient`, and I'm not sure that 1000 bytes is enough

Reply:
I changed buffer type to char and set BUFFER_LEN to 16000